x-pack/test/security_solution_cypress/cypress/tasks/api_calls/exceptions.ts

```diff
@@ -19,15 +19,23 @@ export const createEndpointExceptionList = () =>
   rootRequest<ExceptionListSchema>({
     method: 'POST',
     url: ENDPOINT_LIST_URL,
-    headers: { 'kbn-xsrf': 'cypress-creds', 'x-elastic-internal-origin': 'security-solution' },
+    headers: {
+      'kbn-xsrf': 'cypress-creds',
```

Comment:
`rootRequet` already has this header. Ideally `'x-elastic-internal-origin': 'security-solution'` should be inside `rootRequest` as well.

Reply:
@maximpn I see that this is a broader problem across other API calls referencing `rootRequest`. It is if the cleanup is done in a follow-up PR?